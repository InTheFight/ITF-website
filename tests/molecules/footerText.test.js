import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import FooterText from '../../src/components/molecules/footerText';

test('Displays the footer text', () => {
  const footerText = render(<FooterText />);

  expect(screen.getByRole('heading')).toHaveTextContent('IN THE FIGHT');
  expect(footerText).toMatchSnapshot();
});