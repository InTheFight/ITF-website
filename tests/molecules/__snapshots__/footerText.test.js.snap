// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the footer text 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-gtssRu livFIg"
      >
        <div
          class="sc-dlnjPT jitMvg"
        >
          <h2>
            IN THE FIGHT
          </h2>
          <a
            href="/"
          >
            <img
              alt="ring"
              class="sc-bdnylx hjenXh"
              src=""
            />
          </a>
        </div>
        <p>
          Amem minim mollit non deserunt ullamco est sit aliqua dolor do amet sint.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-gtssRu livFIg"
    >
      <div
        class="sc-dlnjPT jitMvg"
      >
        <h2>
          IN THE FIGHT
        </h2>
        <a
          href="/"
        >
          <img
            alt="ring"
            class="sc-bdnylx hjenXh"
            src=""
          />
        </a>
      </div>
      <p>
        Amem minim mollit non deserunt ullamco est sit aliqua dolor do amet sint.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
